app/models/post.rb

```diff
@@ -16,12 +16,12 @@ class Post < ActiveRecord::Base
     self.body.scan(Haml::Filters::GrowstuffMarkdown::MEMBER_REGEX) do |m|
       # find member case-insensitively and add to list of recipients
       member = Member.where('lower(login_name) = ?', $1.downcase).first
-      recipients << member if member and not recipients.include?(member)
+      recipients << member if member and ! recipients.include?(member)
```

Comment:
Might want to rewrite the logic for these, it reads funny... maybe a .uniq! at the end instead of checking the collection?